packages/erxes-ui/src/utils/core.tsx

```
@@ -787,12 +905,12 @@
   const removeTypename = (obj: any) => {
     if (Array.isArray(obj)) {
       obj.forEach(removeTypename);
-    } else if (typeof obj === 'object' && obj !== null) {
+    } else if (typeof obj === "object" && obj !== null) {
       delete obj.__typename;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)